bye-bye-dead-code/src/main/java/com/dipien/byebyedeadcode/ByeByeDeadCodeExtension.kt

```diff
@@ -29,4 +29,7 @@ open class ByeByeDeadCodeExtension(propertyResolver: PropertyResolver) {
             "src/main/java",
             "src/release/java"
     )
+
+    // Add regex to exclude classes and methods from the report
+    var ignore = listOf<String>()
```

Comment:
Is this name too generic?
Should we have one for classes and other for methods?

Reply:
Done